README.md

```diff
@@ -137,6 +137,255 @@ To download file you should call **GetFile** method
+                  Console.WriteLine($"Short: {updateShort.Update}");
+                  if (updateShort.Update is TLUpdateUserStatus)
+                  {
+                      var status = updateShort.Update as TLUpdateUserStatus;
+                      Console.WriteLine($"User {status.UserId} is {status.Status}");
+                      if (status.Status is TLUserStatusOnline)
+                      {
+                          try
+                          {
+                              var peer = new TLInputPeerUser() { UserId = status.UserId };
+                              client.SendMessageAsync(peer, "Você está online.").Wait();
+                          } 
+			  catch (Exception e)
+              		  {
+                  	      Console.WriteLine(e);
+              		  }
+                      }
+                  }
+              }
+              else if (updates is TLUpdateShortMessage)
+              {
+                  var message = updates as TLUpdateShortMessage;
+                  Console.WriteLine($"Message: {message.Message}");
+                  MarkMessageRead(client, new TLInputPeerUser() { UserId = message.UserId }, message.Id);
+              }
+              else if (updates is TLUpdateShortChatMessage)
+              {
+                  var message = updates as TLUpdateShortChatMessage;
+                  Console.WriteLine($"Chat Message: {message.Message}");
+                  MarkMessageRead(client, new TLInputPeerChat() { ChatId = message.ChatId }, message.Id);
+              }
+              else if (updates is TLUpdates)
+              {
+                  var allUpdates = updates as TLUpdates;
+                  foreach (var update in allUpdates.Updates)
+                  {
+                      Console.WriteLine($"\t{update}");
+                      if (update is TLUpdateNewChannelMessage)
+                      {
+                          var metaMessage = update as TLUpdateNewChannelMessage;
+                          var message = metaMessage.Message as TLMessage;
+                          Console.WriteLine($"Channel message: {message.Message}");
+                          var channel = allUpdates.Chats[0] as TLChannel;
+                          MarkMessageRead(client, 
+                                          new TLInputPeerChannel() { ChannelId = channel.Id, AccessHash = channel.AccessHash.Value }, 
+                                          message.Id );
+                      }
+                  }
+
+                  foreach(var user in allUpdates.Users)
+                  {
+                      Console.WriteLine($"{user}");
+                  }
+
+                  foreach (var chat in allUpdates.Chats)
+                  {
+                      Console.WriteLine($"{chat}");
+                  }
+              }
+          }
+
+          private void MarkMessageRead(TelegramClient client, TLAbsInputPeer peer, int id)
+          {
+	      // An exception happens here but it's not fatal.
```

Comment:
what exception is this exactly? mmm

Reply:
only one i have seen is a Timeout one which is non-fatal

Reply:
will take the comment out

Reply:
well if you've seen a TimeoutException, first paste the `ex.ToString()` here, then replace the catch block to only catch this exception